easybuild/easyconfigs/g/Guile/Guile-2.0.11-foss-2016a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'Guile'
+version = '2.0.11'
+
+homepage = 'https://www.gnu.org/software/guile/'
+description = """Guile is a programming language, designed to help programmers create flexible applications that
+ can be extended by users or other programmers with plug-ins, modules, or scripts."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [('Autotools', '20150215')]
+dependencies = [
+    ('libunistring', '0.9.3'),
+    ('libffi', '3.2.1'),
+    ('gc', '7.4.4', '', ('GCC', '4.9.3-2.25')), 
```

Comment:
@RvDijk any reason you're hard specifying the subtoolchain here? this will cause headaches when people use `--try-toolchain` on this easyconfig...